M2/Macaulay2/d/basictests/factor.m2

```diff
@@ -1,14 +1,14 @@
 assert = x -> if not x then error "assertion failed"
 
-v = Pari$factorint (12 * 2^32 + 3)
-if v === ((2, 3, 5, 13, 41, 61, 1321), (2, 1, 2, 1, 1, 1, 1)) then error "pari/gmp integer conversion: 32-bit reversal"
-assert( v == ((3, 5, 137, 953, 26317), (1, 1, 1, 1, 1)) )
+-- v = Pari$factorint (12 * 2^32 + 3)
+-- if v === ((2, 3, 5, 13, 41, 61, 1321), (2, 1, 2, 1, 1, 1, 1)) then error "pari/gmp integer conversion: 32-bit reversal"
+-- assert( v == ((3, 5, 137, 953, 26317), (1, 1, 1, 1, 1)) )
 
-v = Pari$factorint (1 + 4 * 2^64)
-if v == ((2, 5, 5581, 8681, 49477, 384773), (2, 1, 1, 1, 1, 1)) then error "pari/gmp integer conversion: 64-bit reversal"
-assert( v == ((5, 13, 397, 2113, 312709, 4327489), (1, 1, 1, 1, 1, 1)) )
+-- v = Pari$factorint (1 + 4 * 2^64)
+-- if v == ((2, 5, 5581, 8681, 49477, 384773), (2, 1, 1, 1, 1, 1)) then error "pari/gmp integer conversion: 64-bit reversal"
+-- assert( v == ((5, 13, 397, 2113, 312709, 4327489), (1, 1, 1, 1, 1, 1)) )
```

Comment:
Should these be replaced with current tests?

Reply:
I've removed the file in commit ed1741b30bae13b25aa96e34fb54efe00ba55070.  That test was needed because there was a frequent inconsistency between the internal format of pari  big numbers and ours.